### ISSUE TITLE
how to use the caffe model?

### ISSUE THREAD

**Issue by tingtinglu** [szagoruyko/cvpr15deepcompare#8]
I am new to deep learning, and i only know how to use caffe a little. If i want to use the caffe model to test the pre-trained model, how should i preprocess the input images? or just give two images to the models? Thanks so much !

**Comment by szagoruyko** [comment]
you need to scale images to [0,1] range and subtract mean as in the example here https://github.com/szagoruyko/cvpr15deepcompare/blob/master/torch/extract_cpu.lua#L17

**Comment by OpenPandaLee** [comment]
Hi， Is your issue solved?  using the caffe model of this repository, I can't get the correct output value([-1,1]) ,such as 10.111，-9.433

**Comment by szagoruyko** [comment]
@OpenPandaLee the outputs are not normalized to [-1,1] range by the network, the values you see are expected.

**Comment by OpenPandaLee** [comment]
@szagoruyko thanks four help,  so how can i evaluate similarity of images with the output？
